types/index.d.ts

```diff
@@ -44,6 +46,7 @@ declare namespace fastifyView {
     root?: string;
     viewExt?: string;
     propertyName?: string;
+    asyncPropertyName?: string;
```

Comment:
Can you add a test for this type? We use `tsd`

Reply:
ptal at most recent commit